addon/services/flexberry-logging.js

```diff
@@ -14,22 +12,66 @@ const { getOwner } = Ember;
  */
 export default Ember.Service.extend({
   /**
-    Store for transmit messages to server.
+       Store for transmit messages to the server
+
+       @property {Object} _flexberryStore
+       @private
+       @type DS.Store
+   */
+  _flexberryStore: Ember.inject.service('store'),
+  /**
+       Logger switcher on/off true if remote logging service works correctly
+
+       @property {Boolean} _serverLogEnabled
+       @private
+       @type Boolean
+       @default true
+   */
+  _serverLogEnabled: true,
+  /**
+       Current log level. Set this property to log appropriate interval of log types
+
+       @property {Integer} flexberryLogLevel
+       @type Integer
+       @default 0
+   */
+  flexberryLogLevel: 0,
+
+  /**
+     Log level enumarator
 
-    @property flexberryStore
-    @type DS.Store
-    @default null
+     @property logLevelEnums
+     @type Object
+     @@default { ERROR: 1,WARN: 2, LOG: 3, INFO: 4, DEBUG: 5, DEPRECATION: 6 }
```

Comment:
Two "@".
Is "{ ERROR: 1,WARN: 2, LOG: 3, INFO: 4, DEBUG: 5, DEPRECATION: 6 }" really default value?